lib/js/helpers/classy.js

```diff
@@ -7,26 +7,26 @@ import * as dom from './dom';
 
 // check if an element has a specific class
 export function has (domNode, className) {
-  var elementClass = ' ' + domNode.className + ' ';
-  return elementClass.indexOf(' ' + className + ' ') !== -1;
+  return new RegExp('(\\s|^)' + className + '(\\s|$)').test(domNode.getAttribute('class'));
```

Comment:
This is a regexp that checks if a class exists in the string returned from get attribute. Becuase SVG's dont have a `domNode.class` property that behaves the same as other actuall dom nodes. This lets the classy helpers work for HTML and SVG nodes.